### DISCUSSION TITLE
PrimeNG Browser Support

### DISCUSSION THREAD

**Discussion by DAYANAVM** [primefaces/community#569] [PrimeNG] [answered]
I gone through the PrimeNG documentation and was looking for a browser support . Am unable to find it in the documentation. Could anyone please list the browser support of the Prime NG with versions? Does it work well with ios?

**Comment by onursenture** [comment, marked as answer]
Prime libraries support the same browsers that the JS library depends on. For example, PrimeNG and Angular have the same browser support. https://angular.io/guide/browser-support
